FreeRTOS/FreeRTOS-Plus-TCP
# Misra fix or suppress remaining violations

Fix Remaining Misra violations

Description
-----------
Fix Remaining and new misra violations

Test Steps
-----------
Misra checker + unit tests


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
